build.sbt

```diff
@@ -544,7 +544,7 @@ lazy val junit = project.in(file("test") / "junit")
   .settings(disablePublishing: _*)
   .settings(
     fork in Test := true,
-    libraryDependencies ++= Seq(junitDep, junitIntefaceDep),
+    libraryDependencies ++= Seq(junitDep, junitIntefaceDep, "org.openjdk.jol" % "jol-core" % "0.5"),
```

Comment:
can you extract this into a local at https://github.com/scala/scala/blob/2.12.x/build.sbt#L63-L72 to keep the dependencies together?